python/doc/theory/meta_modeling/polynomial_least_squares.rst

```diff
@@ -3,95 +3,102 @@
-  may be computed using a least squares regression approach. In this
-  context, an experimental design
-  :math:`\underline{\cX} =(x^{(1)},\dots,x^{(N)})`, i.e. a set of
-  realizations of input parameters is required, as well as the
-  corresponding model evaluations
-  :math:`\underline{\cY} =(y^{(1)},\dots,y^{(N)})`.
+A *global* approximation of the model response over its whole
+definition domain is sought. To this end, the coefficients :math:`a_j`
+may be computed using a least squares regression approach. In this
+context, an experimental design that is, a set of observations of
+input parameters, is required:
 
-| The following minimization problem has to be solved:
+.. math::
 
-  .. math::
+    \cX = \left\{ \vect{x}^{(1)}, \dots, \vect{x}^{(N)} \right\},
 
-     \mbox{Find} \quad \widehat{\underline{a}} \quad \mbox{that minimizes} \quad \cJ(\underline{a}) \, \, = \, \, \sum_{i=1}^N \; \left( y^{(i)} \; - \; \underline{a}^{\textsf{T}}  \underline{\psi}(\underline{x}^{(i)}) \right)^2
+as well as the corresponding model evaluations:
 
-  The solution is given by:
+.. math::
 
-  .. math::
+    \cY = \left\{ y^{(1)},\dots,y^{(N)} \right\}.
 
-      \widehat{\underline{a}} \, \, = \, \, \left( \underline{\underline{\Psi}}^{\textsf{T}} \underline{\underline{\Psi}}  \right)^{-1} \; \underline{\underline{\Psi}}^{\textsf{T}}  \; \underline{\cY}
+The following minimization problem has to be solved:
 
-  where:
+.. math::
 
-  .. math::
+    \mbox{Find} \quad \widehat{\vect{a}} \quad \mbox{that minimizes} \quad \cJ(\vect{a})  =  \sum_{i=1}^N \left( y^{(i)} - \vect{a}^{\textsf{T}}  \vect{\psi}\left(\vect{x}^{(i)}\right) \right)^2
+
+Let :math:`\vect{y} = (y^{(1)},\dots,y^{(N)})^T \in \Rset^{N}` be the
+vector of output observations.
+The solution is given by the normal equations:
+
+.. math::
+
+    \widehat{\vect{a}}  =  \left( \boldsymbol{\Psi}^{\textsf{T}} \boldsymbol{\Psi}  \right)^{-1} \boldsymbol{\Psi}^{\textsf{T}}  \vect{y}
+
+where:
+
+.. math::
+
+    \boldsymbol{\Psi}  =  (\psi_{j}\left(\vect{x}^{(i)}\right) , i=1,\dots,N , j = 0,\dots,P-1)
 
-      \underline{\underline{\Psi}} \, \, = \, \, (\psi_{j}(\underline{x}^{(i)}) \; , \; i=1,\dots,N \; , \; j = 0,\dots,P-1)
-
-  It is clear that the above equation is only valid for a full rank
-  information matrix. A necessary condition is that the size :math:`N`
-  of the experimental design is not less than the number :math:`P` of PC
-  coefficients to estimate. In practice, it is not recommended to
-  directly invert
-  :math:`\underline{\underline{\Psi}}^{\textsf{T}} \underline{\underline{\Psi}}`
-  since the solution may be particularly sensitive to an
-  ill-conditioning of the matrix. The least-square problem is rather
-  solved using more robust numerical methods such as *singular value
-  decomposition* (SVD) or *QR-decomposition*.
+It is clear that the above equation is only valid for a full rank
+information matrix. A necessary condition is that the size :math:`N`
```

Comment:
```suggestion
information matrix :math:`\mat{\Psi}`. A necessary condition is that the size :math:`N`
```

Reply:
No: in a regression setting, the information matrix is $\frac{1}{\sigma^2} \Psi^T \Psi$, where $\Psi$ is the design matrix (see [math.stackexchange.com](https://math.stackexchange.com/questions/328835/get-a-fisher-information-matrix-for-linear-model-with-the-normal-distribution-fo)). Equivalently, the covariance matrix is $\sigma^2 (\Psi^T \Psi)^{-1}$. Do you agree on this point?

Reply:
Indeed, you are right! But I think this should be explicitly stated.

Reply:
You suggest to add a content which is related to the linear regression setting, which is different from the least squares approximation setting presented in this page. This is why I think that it is better not to introduce the information matrix here. I send you an email for a more detailed analysis of this issue. Please tell me if that fits your analysis.

Reply:
But it is strange to mention a matrix without being able to write it! Maybe remove the sentence entirely then?

Reply:
Ok, I got it. You are right, the text was confusing: this is the design matrix that needs to be full rank, not the information matrix.